python/run.n

```diff
@@ -1,5 +1,6 @@
 import fek
 import SystemIO
+imp runner
```

Comment:
How would one import files with odd names, or files in folders or in parent folders? How would one import libraries?

I quite like Deno's module system:

```ts
import { export1, export2 } from 'https://deno.land/x/some-module/mod.ts'
import * as namespace from './folder/file-name.ts'
```

This way, the file name can be independent of the code.